codespell_lib/data/dictionary.txt

```diff
@@ -28498,16 +28498,32 @@ traingulates->triangulates
 traingulating->triangulating
 traingulation->triangulation
 traingulations->triangulations
+trainig->training
+trainigs->trainings
+trainling->training
+trainng->training
+trainngs->trainings
```

Comment:
I thought it seemed odd, so suggest both plural and singular:
https://www.englishbusiness.com/blog/training-or-trainings-what-is-the-correct-plural-form/

You may also want to add `trainings->training` to the US to GB dictionary.